enterprise/app/history/history.tsx

```diff
@@ -376,6 +378,7 @@ export default class HistoryComponent extends React.Component {
           )}
         </div>
         {this.props.hash === "#users" && <OrgJoinRequestsComponent user={this.props.user} />}
+        {capabilities.globalFilter && <FilterComponent search={this.props.search} />}
```

Comment:
What do you think about putting this here vs making it feel more "global" - i.e. in the top right where the AW date picker was? I can see pros and cons to each approach.